rotkehlchen/db/upgrades/v39_v40.py

```diff
@@ -334,6 +339,38 @@ def _add_new_tables(write_cursor: 'DBCursor') -> None:
     log.debug('Exit _add_new_tables')
 
 
+def _reset_decoded_events(write_cursor: 'DBCursor') -> None:
+    """
+    Reset all decoded evm events except the customized ones for ethereum mainnet,
+    optimism and polygon.
+    """
+    log.debug('Enter _reset_decoded_events')
+    if write_cursor.execute('SELECT COUNT(*) FROM evm_transactions').fetchone()[0] == 0:
+        return
+
+    customized_events = write_cursor.execute(
+        'SELECT COUNT(*) FROM history_events_mappings WHERE name=? AND value=?',
+        (HISTORY_MAPPING_KEY_STATE, HISTORY_MAPPING_STATE_CUSTOMIZED),
+    ).fetchone()[0]
+    querystr = (
+        'DELETE FROM history_events WHERE identifier IN ('
+        'SELECT H.identifier from history_events H INNER JOIN evm_events_info E '
+        'ON H.identifier=E.identifier AND E.tx_hash IN '
+        '(SELECT tx_hash from_evm_transactions))'
+    )
+    bindings: tuple = ()
```

Comment:
nitpick. I don't think type is needed here. It's infered

```suggestion
    bindings = ()
```

Reply:
lint fails if it isn't there